gammapy/utils/modeling.py

```diff
@@ -34,10 +36,11 @@ class Parameter(object):
     frozen : bool, optional
         Frozen? (used in fitting)
     """
-    __slots__ = ['_name', '_value', '_unit', '_min', '_max', '_frozen']
+    __slots__ = ['_modelname', '_name', '_value', '_unit', '_min', '_max', '_frozen']
 
-    def __init__(self, name, value, unit='', min=np.nan, max=np.nan, frozen=False):
+    def __init__(self, modelname, name, value, unit='', min=np.nan, max=np.nan, frozen=False):
         self.name = name
+        self.modelname = modelname
```

Comment:
Put modelname before name here and in the properties below.
Having everything always in the same order helps to find things and review correctness.